paywall/src/__tests__/middlewares/web3Middleware.test.js

```diff
@@ -543,6 +543,26 @@ describe('Web3 middleware', () => {
     )
   })
 
+  it('should handle ADD_TRANSACTION with data input', () => {
```

Comment:
I think we will need an additional test case here to ensure that the else branch of the conditional is covered

Reply:
The test case right above handles the else case (unmodified because the else case was already being tested)

Reply:
Ah got you, would you mind update the description of the other test to indicate that is for the case where data is not input